test_framework/terraform/aws/sle-micro/user-data-scripts/provision_k3s_agent.sh.tpl

```diff
@@ -18,5 +18,6 @@ elif [ -b "/dev/xvdh" ]; then
   sudo mount /dev/xvdh /var/lib/longhorn
 fi
 
+sudo su -
 curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="agent --token ${k3s_cluster_secret}" K3S_URL="${k3s_server_url}" INSTALL_K3S_VERSION="${k3s_version}" sh -
-sudo systemctl start k3s-agent
+systemctl start k3s-agent
```

Comment:
why sudo is not working but need to switch to root first?

Reply:
What caused the problem is in the `https://get.k3s.io` script. It actually has nothing to do with `systemctl start k3s-agent`.

Reply:
I see. Roger's comment did mention that.